bedrock/settings/base.py

```diff
@@ -680,8 +681,15 @@ def get_app_name(hostname):
     #     uncomment and change this to get more
     #     'num_posts': 20,
     # },
+    "careers": {
+        "url": "https://blog.mozilla.org/careers/",
+        "name": "Life@Mozilla",
+        "num_posts": 20,
+    },
 }
 
+GREENHOUSE_BOARD_TOKEN = config("GREENHOUSE_BOARD_TOKEN", default="mozilla")
```

Comment:
I'm considering changing this to `GREENHOUSE_BOARD`, intentionally removing the `TOKEN` part since this is confusing as it suggests it is a secret, which is confusing.

Reply:
That's a great idea - because that's just the confusion it caused 😄